src/apps/testapps/testBaseCells.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2017-2019 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdlib.h>
+#include "constants.h"
+#include "h3api.h"
+#include "test.h"
+
+SUITE(baseCells) {
+    TEST(getBaseCells) {
+        int count = res0IndexCount();
```

Comment:
This line needs `H3_EXPORT(res0IndexCount)` now.

Reply:
Good catch!